manifests/a/Amazon/Games/latest/Amazon.Games.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Amazon.Games
+PackageVersion: latest
+PackageLocale: en-US
+Publisher: Amazon.com, Inc.
+PublisherUrl: https://www.amazon.com/
+PublisherSupportUrl: https://www.amazon.com/gp/help/customer/contact-us/ref=cu_surl
+PrivacyUrl: https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=508088
+#Author: 
+PackageName: Amazon Games
+PackageUrl: https://gaming.amazon.com/home
+License: Proprietary
+LicenseUrl: https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=508088
+Copyright: 2019-2021, Amazon.com, Inc. or its affiliates
+CopyrightUrl: https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=508088
+ShortDescription: Amazon Games App
```

Comment:
```
ShortDescription: Amazon Games App
```
Please write a better description.

Reply:
The name of the software is "Amazon Games App",

Reply:
e.g. `Amazon Games app is a Windows PC game launcher that allows you to claim, download, and play new and classic Windows PC games every month from Prime Gaming`

Reply:
I'll keep that in mind for my future manifests